maxim/filter_objects.py

```
@@ -1,7 +1,8 @@
 from typing import List, Optional
 
 from .models import RuleGroupType, RuleType
-
+import re
```

Pylint (3.3.7)

[convention] standard import "re" should be placed before local import "models.RuleGroupType"

(C0411)